xarray/test/test_variable.py

```diff
@@ -304,22 +309,23 @@ def test_equals_all_dtypes(self):
     def test_eq_all_dtypes(self):
         # ensure that we don't choke on comparisons for which numpy returns
         # scalars
-        expected = self.cls('x', 3 * [False])
+        expected = self.cls('x', 3 * [False]).to_base_variable()
```

Comment:
Note that you can just use `Variable('x', 3 * [False])` in this case and other ones like it, which is maybe a little clearer.